### PR TITLE
[16.0][MIG] project_task_note: Migration to 16.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Standard migration to v16
- Add resizable options in field note to make it look better
- Change group for label as field inside group appears readonly

TT47764

@Tecnativa

@pedrobaeza @pilarvargas-tecnativa

### CODE REVIEW

**Review by pilarvargas-tecnativa — APPROVED** [review]

File: project_task_note/views/project_task_views.xml [hunk 1]
```diff
@@ -11,14 +11,13 @@
         <field name="inherit_id" ref="project.view_task_form2" />
         <field name="arch" type="xml">
             <field name="description" position="after">
-                <group name="internal_notes_group" string="Internal notes">
+                <label for="notes" />
                     <field
```
Comment (pedrobaeza) [inline project_task_note/views/project_task_views.xml]: Incorrect indentation in this line

Comment (carolinafernandez-tecnativa) [reply project_task_note/views/project_task_views.xml]: fix it

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**